assets/terraform-modules/azure/flatcar-linux/kubernetes/workers/cl/worker.yaml.tmpl

```diff
@@ -7,7 +7,11 @@ systemd:
       mask: true
     - name: iscsid.service
       enable: true
-      enabled: true
```

Comment:
Where is it documented that `enable` should be used instead of `enabled` ?

Reply:
https://coreos.com/ignition/docs/latest/configuration-v2_4.html

Reply:
Not sure if this documentation is a good reference, as it's from CoreOS, however it says that `enable` is deprecated and `enabled` should be used. I think we should remove the other one (and while doing so we can commit it separately :wink:)

Reply:
My bad, good catch. I did exact opposite :facepalm: Fixed it now.